website/docs/r/windows_virtual_machine.html.markdown

```diff
@@ -287,6 +289,16 @@ A `secret` block supports the following:
 
 ---
 
+A `terminate_notification` block supports the following:
```

Comment:
grammer, shouldn't this be
```suggestion
A `termination_notification` block supports the following:
```

and all documentation should probably use termination instead of terminate

Reply:
`linux_virtual_machine_scale_set_resource` and `windows_virtual_machine_scale_set_resource` has it named `terminate_...`,
https://github.com/hashicorp/terraform-provider-azurerm/blob/5361e9cc0772dd69929a47c21575b5be0d832830/internal/services/compute/linux_virtual_machine_scale_set_resource.go#L282
https://github.com/hashicorp/terraform-provider-azurerm/blob/5361e9cc0772dd69929a47c21575b5be0d832830/internal/services/compute/windows_virtual_machine_scale_set_resource.go#L304
However, `orchestrated_virtual_machine_scale_set_resource` has it named `termination_...`
https://github.com/hashicorp/terraform-provider-azurerm/blob/5361e9cc0772dd69929a47c21575b5be0d832830/internal/services/compute/orchestrated_virtual_machine_scale_set_resource.go#L185

Since the property name in API is called `TerminateNotificationProfile`, maybe it's better to use `terminate_...`, I'll start another change to align the name in `orchestrated_virtual_machine_scale_set_resource`.

Reply:
Hi @katbyte I've created another change try renaming the property name #15570

Reply:
Update
--------
I've updated the property name from `terminate_notification` to `termination_notification` according to the suggestion